### ISSUE TITLE
Check page stats counter

### ISSUE THREAD

**Issue by bep** [spf13/hugo#2757]
See https://discuss.gohugo.io/t/hugo-0-18-release-scheduled-for-mon-19th-december-please-help-test-test-test/4726/2

**Comment by bep** [comment]
This is correct, as it now includes the old node pages, taxonomies etc. So it is more correct than before.

**Comment by kaushalmodi** [comment]
Would it be possible to re-add the content page count?

---

I have pasted below from [this post](https://discuss.gohugo.io/t/hugo-0-18-release-scheduled-for-mon-19th-december-please-help-test-test-test/4726/8?u=kaushalmodi) on that same discussion thread: 

> 0 draft content
> 0 future content
> 0 expired content
> 29 pages created
> 0 non-page files copied
> 0 paginator pages created
> 9 tags created
> 5 topics created
> total in 35 ms

> That 29 pages matches the count of 
> *  1 home
> *  9 content
> *  3 sections (post + project + presentation)
> *  9 tags 
> *  1 at tag root
> *  5 topics
> *  1 at topic root

Based on above, the count of 29 already includes the "9 tags" and "5 topics" pages, whereas those counts are printed independently too. 

I found the "content page count" as a very useful feedback. Actually that was the main number I paid attention to when publishing.

So while the total count of 29 is kept, can the content page count be re-added as well?

**Comment by bep** [comment]
>So while the total count of 29 is kept, can the content page count be re-added as well?

Everything is possible, but that is another issue (which someone will have to create). I just closed this as "not a bug".
